process/dd-review-template.md

```diff
@@ -24,19 +24,19 @@ As a secondary goal, it is in the interests of the broader CNCF ecosystem that t
 /* Use appropriate Section */
 
 ### Sandbox Graduation (Exit Requirements)
-1. Document that it is being used successfully in production by at least three independent end users which with focus on adequate quality and scope defined.
+1. Document that it is being used successfully in production by at least three independent direct adopters which with focus on adequate quality and scope defined.
```

Comment:
Is there a definition of a direct adopter? 

https://github.com/cncf/toc/blob/main/FAQ.md#what-is-the-definition-of-an-adopter did not have that in the taxonomy of words.

Reply:
added to FAQs.